src/main/java/core/basesyntax/CircleSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class CircleSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();
+    private static final ColorSupplier color = new ColorSupplier();

```

Comment:
The 'color' variable can be not static, because it's not shared between different instances of the 'CircleSupplier' class.